libraries.json

```
@@ -13,53 +19,212 @@
 		"image": "",
 		"language": "Python"
 	},
+
+	"Framework7": {
+		"imports": ["framework7"],
+		"technologies": ["Mobile applications", "Desktop applications", "Progressive Web Apps", "CSS frameworks"],
+		"description": "A free and open source framework to develop mobile, desktop or web apps with native look and feel",
+		"language": "JavaScript"
+	},
+	"Go Micro": {
+		"imports": ["github.com/micro/go-micro"],
+		"technologies": ["microservice"],
+		"description": "Go Micro provides the core requirements for distributed systems development including RPC and Event driven communication.",
+		"language": "Go"
+	},
+	"Gulp": {
+		"imports": ["gulp"],
+		"technologies": ["Build tools"],
+		"description": "Gulp is an open-source JavaScript toolkit created by Eric Schoffstall used as a streaming build system in front-end web development.",
+		"language": "JavaScript"
+	},
+	"Ionic": {
+		"imports": ["@ionic/"],
+		"technologies": ["Mobile applications"],
+		"description": "Ionic is the app development platform for web developers",
+		"language": "JavaScript"
+	},
+	"Jest": {
+		"imports": ["jest"],
+		"technologies": ["Testing"],
+		"description": "Jest is a delightful JavaScript Testing Framework with a focus on simplicity",
+		"language": "JavaScript"
+	},
+	"Materialize CSS": { 
+		"imports": ["materialize-css"],
+		"technologies": ["CSS frameworks"],
+		"description": "A modern responsive front-end framework based on Material Design",
+		"language": "JavaScript"
+	},
 	"MeteorJS": {
 		"imports": ["meteor"],
 		"technologies": ["Web Development", "SPA", "Full Stack"],
 		"description": "Meteor.js is a full stack framework. It means that you can control everything from the database to the UI with the same language and the same framework, and it is one of Meteor’s primary advantages.",
 		"image": "https://user-images.githubusercontent.com/841294/26841702-0902bbee-4af3-11e7-9805-0618da66a246.png",
 		"language": "JavaScript"
 	},
+	"MobX": { 
+		"imports": ["mobx"],
+		"technologies": ["Web Development", "SPA"],
+		"description": "MobX is a battle tested library that makes state management simple and scalable by transparently applying functional reactive programming (TFRP)",
+		"language": "JavaScript"
+	},
+	"NativeScript": {
+		"imports": ["nativescript"],
+		"technologies": ["Mobile applications"],
+		"description": "Open source framework for building truly native mobile apps with Angular, Vue.js, TypeScript, or JavaScript.",
+		"language": "JavaScript"
+	},
+	"NextJS": {
+		"imports": ["nextjs"],
+		"technologies": ["Web Development", "SPA", "Server side rendering"],
+		"description": "Production grade React applications that scale. The world's leading companies use Next.js to build server-rendered applications, static websites, and more.",
+		"language": "JavaScript"
+	},
+	"NgRx": { 
+		"imports": ["@ngrx/core"],
```

```suggestion
		"imports": ["@ngrx/"],
```

The exact package `@ngrx/core` is not used anymore, most likely stuff is imported from `@ngrx/store` and `@ngrx/effects` (see https://ngrx.io/guide/store for the names of all of the packages) so I'd suggest changing it to just `@ngrx/`